pkg/build/client/clients.go

```diff
@@ -0,0 +1,69 @@
+package client
+
+import (
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	osclient "github.com/openshift/origin/pkg/client"
+)
+
+// BuildConfigGetInterface provides methods for getting BuildConfigs
```

Comment:
Consider me mollified on the interfaces here.

It's probably not good to call these "Interface" - instead "Getter" or "Updater" is more appropriate.